variable/creation.cpp

```diff
@@ -62,4 +62,6 @@ Variable special_like(const Variable &prototype, const FillValue &fill) {
   throw std::runtime_error("Unsupported fill value.");
 }
 
+Variable zero_like(const Variable &var) { return {var, Dimensions{}}; }
```

Comment:
Maybe a docstring to explain that this is singular/scalar intentionally and should *not* be refactored to to what `zeros_like` does?